outputs.tf

```diff
@@ -21,6 +21,13 @@ output "iam_role" {
   value       = local.create_iam_role ? module.iam_role[0].arn : var.iam_role_arn
 }
 
+output "iam_role_organization_aggregator_arn" {
+  description = <<-DOC
+  IAM Role the Aggregator uses to read organization data
+  DOC
```

Comment:
When you have a short one-line description, consider just using quotes rather than a HEREDOC.